stdlib/public/Glibc/Glibc.swift

```diff
@@ -29,6 +29,100 @@ public var errno: Int32 {
 // fcntl.h
 //===----------------------------------------------------------------------===//
 
+@warn_unused_result
+@_silgen_name("_swift_Glibc_open")
+func _swift_Glibc_open(path: UnsafePointer<CChar>,
+  _ oflag: CInt,
+  _ mode: mode_t
+) -> CInt
+
+@warn_unused_result
+@_silgen_name("_swift_Glibc_openat")
+func _swift_Glibc_openat(
+  fd: CInt,
+  _ path: UnsafePointer<CChar>,
+  _ oflag: CInt,
+  _ mode: mode_t
+) -> CInt
+
+@warn_unused_result
+public func open(
+  path: UnsafePointer<CChar>,
+  _ oflag: CInt
+) -> CInt {
```

Comment:
Just wondering: why not use a default value of `0` for the `mode: mode_t` parameter instead of duplicating those functions?